committees/tests/test_committee_meeting_detail_view.py

```diff
@@ -302,3 +319,21 @@ def test_create_tag(self):
         self.assertEqual(res.status_code, 200)
         self.new_tag = Tag.objects.get(name='new tag')
         self.assertIn(self.new_tag, self.meeting_1.tags)
+
+    @waffle_testutils.override_flag('show_member_presence', active=True)
+    def test_meeting_has_show_presence_in_context_when_flag_is_set(
+            self):
+        res = self.client.get(self.meeting_1.get_absolute_url())
+        self.assertEqual(res.status_code, 200)
+        self.assertTrue(res.context['show_member_presence'],
+                        msg="context property 'show_member_presence'"
+                            " was not set to True.")
+
+    @waffle_testutils.override_flag('show_member_presence', active=False)
+    def test_meeting_does_not_show_presence_in_context_when_flag_is_unset(
+            self):
+        res = self.client.get(self.meeting_1.get_absolute_url())
+        self.assertEqual(res.status_code, 200)
```

Comment:
Same comment as above., better to use verify and given to encapsulate setup and asserts